nx.json

```diff
@@ -29,10 +29,22 @@
       "inputs": ["default", "^production"]
     },
     "pb-ve-env-setup": {
-      "cache": true
-    },
-    "nx-release-publish": {
+      "cache": true,
+      "inputs": [
+        "{projectRoot}/project.json",
+        {
+          "runtime": "node --version"
+        },
+        {
+          "runtime": "npm --version"
+        },
+        {
+          "externalDependencies": ["verdaccio"]
+        },
+        "^production"
+      ]
```

Comment:
Imo this should be placed in the plugin directly.

Reply:
Indeed, but at the nx.json level we can declare custom namedInputs which might be more appropriate than `production`. See https://github.com/push-based/nx-verdaccio/pull/48#discussion_r1797335671

Reply:
Got it!! Lets see what they say...